test/house_test.rb

```diff
@@ -0,0 +1,88 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < MiniTest::Test
+  def setup
+    @house = House.new('$400000', '123 sugar lane')
+    @room1 = Room.new(:bedroom, 10, '13')
+    @room2 = Room.new(:bedroom, 11, '15')
+    @room3 = Room.new(:living_room, 25, '15')
+    @room4 = Room.new(:basement, 30, '41')
+  end
+
+  def test_it_exists_with_attributes
+    assert_instance_of House, @house
+    assert_equal 400_000, @house.price
+    assert_equal '123 sugar lane', @house.address
+  end
+
+  def test_it_starts_no_rooms_and_can_add_rooms
+    assert_equal [], @house.rooms
+    @house.add_room(@room1)
+    @house.add_room(@room2)
+    assert_equal [@room1, @room2], @house.rooms
+  end
+
+  def test_it_can_determine_market_average
+    refute @house.above_market_average?
```

Comment:
remember that if you have multiple possible outcomes, you should test them all. i'd expect to see another house that was above market average, and the relevant assertion for that.